wordpress-mobile/WordPress-iOS
# Issue/show media uploading state better

This PR implements in the Aztec the Navigation Bar button to indicate that Upload Media is happening, and allows the cancelation of all uploads when pressing it.

To test:
 - Start the app
 - Make sure Aztec is active
 - Start a new post
 - Add some media to the post
 - Check that while the uploads are going the blog picker/name is replaced with a message saying Media Uploading with an activity indicator in front.
 - Check that the message goes away after the uploads are done.
 - Tap to add other media
 - While the upload is going tap on the Media Uploading message.
 - See if an alert is displayed and if you press Yes all pending uploads are canceled and removed from the post.

On the process of fixing it, I noticed that the original hybrid editor was also not showing the intended message so I also fixed that.
